index.html

```diff
@@ -1,227 +1,249 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Microverse Porfolio Setup</title>
-    <link rel="stylesheet" href="main.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  </head>
-  <body>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Microverse Porfolio Setup</title>
+  <link rel="stylesheet" href="main.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+
+  <header>
+    <div class="main-header">
+      <div class="Logo">
+        <a href="#">My Logo</a>
+      </div>
+
+      <nav>
+        <div class="bar-bar">
+          <ul>
+            <li><a href="">Porfolio</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contact Us</a></li>
+          </ul>
+
+          <a href="javascript:void(0);" class="openMenu icon">
+            <i class="fa fa-bars"></i>
+          </a>
+        </div>
+      </nav>
+
+    </div>
    
-    <header>
-      <div class="main-header">
-          <div class="Logo">
-            <a href="#">My Logo</a>
-          </div>
-          
-            <nav>
-              <div class="bar-bar">
-                <ul>
-                  <li><a href="">Porfolio</a></li>
-                  <li><a href="">About</a></li>
-                  <li><a href="">Contact Us</a></li>
-                </ul>
-
-               
-                <a href="javascript:void(0);" class="icon" onclick="myFunction()">
-                  <i class="fa fa-bars"></i>
-                </a>
-              </div>
-            </nav>
+
+    <div class="mobile-links">
+      <a href="#" class="closeMenu"><i class="fa fa-times closeMenu" aria-hidden="true"></i></a>
+      <ul>
+        <li><a href="#">Porfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact Us</a></li>
+        <li><a href="#">For Mobile</a></li>
+      </ul>
+    </div>
+  </header>
```

Comment:
- [ ] Kindly the mobile menu doesn't show, there's no hamburger button so I can click on it.

![image](https://user-images.githubusercontent.com/35535130/167514399-5b801b5c-d234-4480-9d24-837fc96c813e.png)